### PR TITLE
Update popup.html

### PR DESCRIPTION
**Pull request by MWB6763**
Resolves #

### CODE REVIEW

File: popups/cloud-games/popup.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <script src="../../webpages/popup-loader.js" type="module"></script>
   </head>
 
-  <body v-cloak>
+  <body v-cloak style="user-select:none">
```
Comment (CST1229) [inline popups/cloud-games/popup.html:18]: You should modify `popup.css` instead of adding inline styles

**Comment by CST1229** [comment]
Also, please explain the reason for your changes.

**Comment by apple502j** [comment]
Closing - I am sure there are many places that require them, so we'll revisit and file a PR addressing everything.

File: popups/cloud-games/popup.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
       <div class="games" v-for="project of projectsSorted">
         <div class="title">
           <a class="project-name nolink" :href="'https://scratch.mit.edu/projects/' + project.id" target="_blank"
-            >{{ project.title }}</a
+            title="{{ project.title }}">{{ project.title }}</a
```
Comment (RedGuy12) [inline popups/cloud-games/popup.html:30]: lol this doesnt even work

**Pull request closed without merging**